Cubical/Structures/Poset.agda

```diff
@@ -16,45 +18,83 @@ open import Cubical.Data.Sigma.Properties
 -- set by ℓ₀ and the level of the relation result by ℓ₁.
 private
   variable
-    ℓ₀ ℓ₁ : Level
+    ℓ ℓ₀ ℓ₁ ℓ₀′ ℓ₁′ ℓ₀′′ ℓ₁′′ : Level
 
 Order : (ℓ₁ : Level) → Type ℓ₀ → Type (ℓ-max ℓ₀ (ℓ-suc ℓ₁))
 Order ℓ₁ A = A → A → hProp ℓ₁
 
-order-iso : (M N : Σ (Type ℓ₀) (Order ℓ₁)) →  fst M ≃ fst N → Type (ℓ-max ℓ₀ ℓ₁)
-order-iso (A , _⊑₀_) (B , _⊑₁_) (f , _) = (x y : A) → [ x ⊑₀ y ⇔ f x ⊑₁ f y ]
-
 isSetOrder : (ℓ₁ : Level) (A : Type ℓ₀) → isSet (Order ℓ₁ A)
 isSetOrder ℓ₁ A = isSetΠ2 λ _ _ → isSetHProp
 
-Order-is-SNS : SNS {ℓ₀} (Order ℓ₁) order-iso
-Order-is-SNS {ℓ₁ = ℓ₁} {X = X}  _⊑₀_ _⊑₁_ = f , f-equiv
+-- We first start by defining what it means a for a function to be
+-- order-preserving. The name "monotonic" is reserved for partial orders.
+
+isOrderPreserving : (M : Σ (Type ℓ₀) (Order ℓ₁)) (N : Σ (Type ℓ₀′) (Order ℓ₁′))
```

Comment:
Wouldn't `M` and `N` look a bit nicer if you used `TypeWithStr` (https://github.com/agda/cubical/blob/master/Cubical/Foundations/Structure.agda#L14)?

Reply:
Yes, I should get into a habit of using `TypeWithStr`.

Reply:
Addressed in abd3a7f9867f495a0b5c3a259cf24984727439be.